frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -170,4 +500,18 @@ export type UninstallOperatorModalProps = {
   csv?: ClusterServiceVersionKind;
 };
 
+type OperandsTableProps = {
+  operands: K8sResourceCommon[];
+  loaded: boolean;
+  csvName: string;
+  cancel?: () => void;
+};
+
+type operandErrorsProps = { operand: K8sResourceCommon; errorMessage: string };
```

Comment:
I wouldn't call this props since it's not a components. Types should have a uppercase first letter.

```suggestion
type OperandError = { operand: K8sResourceCommon; errorMessage: string };
```

Reply:
fixed